docs/usage/Check-security-updates.md

```diff
@@ -0,0 +1,9 @@
+Check security updates
+======================
```

Comment:
Use `#` for creating headings in `*.md`. See https://guides.github.com/features/mastering-markdown